README.md

```diff
@@ -103,6 +103,32 @@ Sample output:
 ConfigurationArn: arn:aws:cloudformation:us-east-1:000000000000:type-configuration/hook/AWS-CloudFormation-SampleHook/default
 ```
 
+## Experimental Commands
+
+To enable experimental commands: you will need to set the environment variable `CFN_CLI_HOOKS_EXPERIMENTAL` to `enabled`. Example for the Bash shell: `export CFN_CLI_HOOKS_EXPERIMENTAL=enabled`.
+
+### Command: enable-lambda-function-invoker
+
+To activate and set the type configuration of the `AWSSamples::LambdaFunctionInvoker::Hook`third-party [hook](https://github.com/aws-cloudformation/aws-cloudformation-samples/tree/main/hooks/python-hooks/lambda-function-invoker) in your AWS account, use the `enable-lambda-function-invoker` command.
+
+This hook will invoke the Lambda function that is passed to the `--lambda-function-arn` argument. Optionally, `--failure-mode`, `--execution-role-arn`, `--alias`, and `--include-targets` can all be specified with the following behavior:
+
+- `--failure-mode` changes the failure mode to either `FAIL` or `WARN` (Default is `FAIL`).
+- `--execution-role-arn` changes the execution role for this hook to use during runtime (Default is the role used to activate the type).
+- `--alias` changes the type name for this hook in your account. For example, this can be used to change `AWSSamples::LambdaFunctionInvoker::Hook` to `MyCompany::MyOrganization::S3BucketCheckHook`. Note: If `alias` is specified, `execution-role-arn` must also be specified.
```

Comment:
```suggestion
- `--alias` changes the type name for this hook in your account and in a given AWS region. For example, this can be used to change `AWSSamples::LambdaFunctionInvoker::Hook` to `MyCompany::MyOrganization::S3BucketCheckHook`. Note: when you use `--alias`, you still need to use `--execution-role-arn` to specify an existing execution role for the hook.
```